lib/ansible/modules/remote_management/foreman/katello.py

```diff
@@ -226,8 +226,12 @@ def manifest(self, params):
         try:
             file = open(os.getcwd() + params['content'], 'r')
```

Comment:
A much cleaner approach is to use `with`:

```python
with open(os.getcwd() + params['content'], 'r') as content_fp:
    content = content_fp.read()
```

This removes the ugly finally with if. It also doesn't override the built in function `file`. Then you could do a try/except around that:

```python
try:
    with open(os.getcwd() + params['content'], 'r') as content_fp:
        content = content_fp.read()
except Exception as e:
    self._module.fail_json(msg="Manifest import failed with %s" % to_native(e), exception=traceback.format_exc())
```